cymetric/metrics.py

```diff
@@ -230,6 +230,38 @@ def agents(series):
 del _agentsdeps, _agentsschema
 
 
+# Transaction Masses [kg]
+_transdeps = [
+    ('Materials', ('SimId', 'ResourceId', 'ObjId', 'TimeCreated'), 'Mass'), 
+    ('Transactions', ('SimId', 'TransactionId', 'SenderId', 'ReceiverId', 
+        'ResourceId'), 'Commodity')
+    ]
+
+_transschema = [
+    ('SimId', ts.UUID), ('TransactionId', ts.INT), 
+    ('ResourceId', ts.INT), ('ObjId', ts.INT), 
+    ('TimeCreated', ts.INT), ('SenderId', ts.INT), 
+    ('ReceiverId', ts.INT), ('Commodity', ts.STRING), 
+    ('Mass', ts.DOUBLE)
+    ]
+
+@metric(name='TransactionQuant', depends=_transdeps, schema=_transschema)
+def transaction_quant(series):
+    """TransQuant metric returns the quantity of each transaction throughout 
+    the simulation.
+    """
+    trans_index = ['SimId', 'TransactionId', 'ResourceId', 'ObjId', 
+            'TimeCreated', 'SenderId', 'ReceiverId', series[1].name]
+    trans = pd.merge(series[0].reset_index(), series[1].reset_index(),
+            on=['SimId', 'ResourceId'], how='inner').set_index(trans_index)
+    trans = trans.groupby(level=trans_index)['Mass'].sum()
```

Comment:
`Mass` is only a significant term for material resources. I think this should probably be more general (for all transactions). Perhaps a general resource quantity metric would be useful as well? I'll let you think about what's best. There's no rush on this =)